cmux-tui/scripts/repro-10431-dash-only.py

```
@@ -0,0 +1,146 @@
+import pty
+import select
+import signal
+import sys
+import time
+
+ITERS = int(os.environ.get("REPRO_ITERS", "30"))
+LOAD = int(os.environ.get("REPRO_LOAD", str(os.cpu_count() or 4)))
+WRITE_CHUNK = int(os.environ.get("REPRO_WRITE_CHUNK", "1"))
+ECHO_DELAY = float(os.environ.get("REPRO_ECHO_DELAY", "0"))
+ECHO_READ = int(os.environ.get("REPRO_ECHO_READ", "65536"))
+CAPTURE = "/tmp/paste-cap-dash.txt"
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/paste-cap-dash.txt"

(S108)